editoast/src/views/authz.rs

```diff
@@ -46,22 +148,16 @@ enum AuthzError {
     Authorizer(#[from] AuthorizerError),
     #[error("Authorization error")]
     Authz(#[from] AuthorizationError),
+    #[error("Bad grants operation : {message}")]
+    BadGrantsOperation { message: String },
```

Comment:
```suggestion
    SimultaneousGrantsAndRevokes,
```

let's be precise + the `#[error(MESSAGE)]` defines the message that's shown, the content is for context data usually